### PR TITLE
Update lock files and minimum typescript version

### PR DESCRIPTION
**Pull request by edwardlee-msft**
# What

Update lock files
Update controlBar style that was impacted due to updated lock files
Update minimum typescript version to be azure minimum version compliant and to work with new 22 node version

# Why

To address Dependabot alert for http-proxy-middleware

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by vhuseinova-msft — APPROVED** [review]

File: packages/react-components/src/components/styles/ControlBar.styles.ts [hunk 1]
```diff
@@ -128,8 +128,7 @@ export const controlButtonStyles: IButtonStyles = {
     border: 'none'
   },
   flexContainer: {
-    flexFlow: 'column',
-    display: 'contents'
+    display: 'unset'
```
Comment (emlynmac) [inline packages/react-components/src/components/styles/ControlBar.styles.ts:131]: Is this change intentional for the lock file updates?

Comment (edwardlee-msft) [reply packages/react-components/src/components/styles/ControlBar.styles.ts:131]: yes, it is intentional. The update lock files requires this change.

File: common/config/rush/common-versions.json [hunk 2]
```diff
@@ -66,7 +66,7 @@
       "5.3.3",
 
       // Typescript check project's dependency -- this is our minimum supported typescript version of our npm package.
-      "4.6.3"
+      "4.8.4"
```
Comment (JamesBurnside) [inline common/config/rush/common-versions.json:69]: We should update our storybook docs also with this new version

File: packages/react-components/src/components/styles/ControlBar.styles.ts (hunk 1 above)
Comment (edwardlee-msft) [reply packages/react-components/src/components/styles/ControlBar.styles.ts:131]: All visual regression tests are passing. So we are good. Also from testing locally, things are looking good.

**Review by JamesBurnside — APPROVED** [review]
